db/seeds.rb

```diff
@@ -9,3 +9,9 @@
 #   movies = Movie.create([{ name: 'Star Wars' },
 #   { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Initialize first account:
+User.create! do |u|
```

Comment:
write it in 1 line
User.create!(email: 'sdfdsf@email.com', password: '123456', password_confirmation: '123456')